roles/baseline/defaults/main.yml

```diff
@@ -31,15 +31,15 @@ METRICS_SERVER_CONFIG:
 ## Ingress-nginx - Defaults
 ingressVersions:
   k8sMinorVersionCeiling:
-    value: 21
-    api:
-      chartVersion: 3.40.0
-      appVersion: 0.50.0
-  k8sMinorVersionFloor:
-    value: 22
+    value: 23
     api:
       chartVersion: 4.3.0
```

Comment:
I see this is a standard but why do we have the `appVersion` along with the `chartVersion`? The `chartVersion` will control the application version being installed. Seems like extra work.

Reply:
Good catch, I agree that `chartVersion` should take care of what application version is being installed, but it's been present in code for a while so I'll check if the `appVersion` is used anywhere else in the code?

Reply:
@thpang the appVersion is not necessarily required and could be removed without consequence.

When this block was initially implemented I added it as documentation so a user/dev could easily lookup the version in the [ingress-kubernetes compatibility table](https://github.com/kubernetes/ingress-nginx#supported-versions-table) without having to jump into the helm index yaml to determine what the app version is.
See this comment: https://github.com/sassoftware/viya4-deployment/pull/181#discussion_r791961151

Reply:
I would suggest from a support perspective we remove this in the future.